src/main/java/appeng/client/render/items/ToolBiometricCardRender.java

```diff
@@ -150,6 +153,7 @@ public void renderItem( final ItemRenderType type, final ItemStack item, final O
 				tessellator.addVertexWithUV( x, y + 1, z, u, v );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Cast one of the operands of this addition operation to a "double". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2184)